mne_bids/read.py

```diff
@@ -256,6 +257,53 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
     return raw
 
 
+def get_matched_empty_room(bids_fname, bids_root):
+    """Get matching empty room file.
+
+    Parameters
+    ----------
+    bids_fname : str
+        The filename for which to find the matching empty room file.
+    bids_root : str
+        Path to the BIDS root folder.
+
+    Returns
+    -------
+    er_fname : str | None.
+        The filename corresponding to the empty room.
+        Returns None if no file found.
+    """
+    from mne_bids.write import make_bids_folders
+
+    bids_fname = op.basename(bids_fname)
+    _, ext = _parse_ext(bids_fname)
+
+    raw = read_raw_bids(bids_fname, bids_root)
+    if raw.info['meas_date'] is None:
+        raise ValueError('Measurement date not available. Cannot get matching'
+                         'empty room file')
+
+    ref_date = datetime.fromtimestamp(raw.info['meas_date'][0])
+    search_path = make_bids_folders(output_path=bids_root, subject='emptyroom',
+                                    session='**', make_dir=False)
+    search_path = op.join(search_path, '**', '**%s' % ext)
+    er_fnames = glob.glob(search_path)
+
+    if len(er_fnames) == 0:
+        return None
+
```

Comment:
Maybe more pythonic to define `best_er_fname=None` before the loop. If there's nothing to loop, will just return None.